src/redux/categories/categories.js

```diff
@@ -0,0 +1,36 @@
+// action types
+const STATUS = 'status';
```

Comment:
- [ ] As stated before, Kindly consider the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux).